src/Middleware/WorkerMiddleware.php

```diff
@@ -49,23 +47,11 @@ class WorkerMiddleware implements MiddlewareInterface
     private $container;
 
     /**
-     * Map message names to a middleware name. For instance:
-     * [
-     *     'image.saved' => ProcessImageMiddleware::class
-     * ]
-     *
-     * @var array
-     */
-    private $messagesMapping;
-
-    /**
-     * @param array              $messagesMapping
      * @param ContainerInterface $container
      */
-    public function __construct(array $messagesMapping, ContainerInterface $container)
+    public function __construct(ContainerInterface $container)
```

Comment:
That's a BC break. We should use this param to instantiate the `WorkerMessageRouterMiddleware` internally.